posts/atomic-design-bootstrap.md

```diff
@@ -0,0 +1,154 @@
+---
+authors:
+- Anthony Rey
+tags:
+- Atomic Design
+- Front-End
+- Bootstrap
+date: 2020-01-16T15:09:00.000Z
+title: "The cohabitation between Atomic Design and Bootstrap"
+image: https://raw.githubusercontent.com/gnuk/blog-usa/atomic-bootstrap/images/2020/01/atomic-design-bootstrap.png
+---
+
+My previous [blog post](https://blog.ippon.tech/atomic-design-in-practice/) was about Atomic Design in practice. I used this organisational method for several years in many teams and different projects.
+
+At the same time, often, colleagues are promoting [Bootstrap](https://getbootstrap.com) (or equivalent) usage to make easier the graphical interfaces integration on the web. In addition, a large number of tools and frameworks integrate it natively.
+
+However, it is difficult to use Bootstrap in an Atomic Design approach. We quickly lose the consistency of our application in favor of Bootstrap's graphical identity.
+
+I want to share with you my experience of _Bootstrap_ so that you know how to make it cohabit with your own styles rather than fighting against it.
```

Comment:
```suggestion
I want to share with you my experience with _Bootstrap_ so that you know how to make it coexist with your own styles rather than fighting against it.
```